ckanext/dcat/tests/profiles/dcat_ap_2/test_scheming_support.py

```diff
@@ -839,6 +838,116 @@ def test_statement_literal(self):
         assert dataset["notes"] == "This is a dataset"
         assert dataset["access_rights"] == "Some statement"
 
+    def test_multiple_contacts(self):
+
+        data = """
+        @prefix dcat: <http://www.w3.org/ns/dcat#> .
+        @prefix dct: <http://purl.org/dc/terms/> .
+        @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+        @prefix vcard: <http://www.w3.org/2006/vcard/ns#> .
+
+        <https://example.com/dataset1>
+          a dcat:Dataset ;
+          dct:title "Dataset 1" ;
+          dct:description "This is a dataset" ;
+            dcat:contactPoint [ a vcard:Kind ;
+                vcard:fn "Test Contact 1" ;
```

Comment:
Could you for example add  v:hasUID <https://orcid.org/0000-0002-9095-9201> ;  for the different contacts?  Then you know also that is working